core/src/main/java/com/netflix/conductor/core/index/NoopIndexDAO.java

```diff
@@ -160,4 +160,9 @@ public List<String> searchArchivableWorkflows(String indexName, long archiveTtlD
     public long getWorkflowCount(String query, String freeText) {
         return 0;
     }
+
+    @Override
+    public SearchResult<WorkflowSummary> getSummaries(SearchResult<String> searchResultIds) {
```

Comment:
It is good practice to return Unsupported Exception, but it is even more important that the style of methods is the same (either all methods return Unsupported Exception or all methods should log the error and return a negative value).